homeassistant/components/gios/config_flow.py

```diff
@@ -0,0 +1,97 @@
+"""Adds config flow for GIOS."""
+import logging
+
+from async_timeout import timeout
+from gios import Gios, NoStationError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import CONF_STATION_ID, DEFAULT_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@callback
+def configured_instances(hass, condition):
+    """Return a set of configured GIOS instances."""
+    return set(
+        entry.data[condition] for entry in hass.config_entries.async_entries(DOMAIN)
+    )
```

Comment:
You can use the outcome from this PR https://github.com/home-assistant/home-assistant/pull/29806 to achieve this

Reply:
@Kane610 Done